README.md

```diff
@@ -1129,4 +1131,49 @@ List of samples:
 
  - Invoke run-sample-import.ps1
 
+<br><br>
+
+---
+# Performance Best Practices
+
+## Import Job Settings
+
+### Encoding
+For improved performance when dealing with fileshare data on ADLS, we highly recommend encoding files in UTF-16. By doing so, you can avoid the need for conversion to the correct encoding, leading to significant time savings in your document and image workflows.
```

Comment:
Maybe is worth to mention that this refers to extracted text or other long text files. (not load file)
"...we highly recommend using extracted text or other long text files encoded in UTF-16"

Reply:
done